core/include/mmcore/param/FilePathParam.h

```diff
@@ -27,18 +27,20 @@ namespace param {
  */
 class FilePathParam : public AbstractParam {
 public:
+    enum FilePathFlagsInternal_ : uint32_t { Internal_NoExistenceCheck = 1 << 2, Internal_RestrictExtension = 1 << 3 };
```

Comment:
Shouldn't we just keep them in the original FilePathFlags_ enum?
The splitup would only make sense if we could make this separate enum private, but these are still required by some other classes, so they are probably required to stay public? Further as we are only using an enum here not enum class, the public interface of the FilePathParam does not change by splitting them, users still can just do `FilePathParam::Internal_NoExistenceCheck`. Also for refactoring in future it may is unexpected to have definition of bit sets separated over multiple enums, which could then lead to double definition of bits if someone is not careful.

Reply:
Good point. See commit 0b0b1cd ...